R/dbSendQuery.R

```diff
@@ -28,11 +28,15 @@ NULL
 .dbSendQuery <- function(conn, statement, ...) {
   url <- paste0(conn@host, ':', conn@port, '/v1/statement')
   status <- 503
+  retries <- 3
   headers <- .request.headers(conn)
-  while (status == 503) {
+  while (status == 503 || (retries > 0 && status >= 400)) {
```

Comment:
Can we be more defensive in the `status >= 400` check in case new codes ever get added to the standard (seems unlikely but just in case). Maybe change it to `status >= 400 && status < 600`